demos/serverless/deploy.js

```diff
@@ -48,6 +49,7 @@ function usage() {
   console.log(`  -l, --disable-printing-logs          Disable printing logs`);
   console.log(`  -o, --capture-output-prefix          Prefix for S3 bucket name`);
   console.log(`  -i, --opt-in-regions                 Comma separated list of additional opt-in regions to enable for media capture`);
+  console.log(`  -m, --chime-sdk-meetings-endpoint    AWS SDK Chime Meetings endpoint`);
```

Comment:
Suggest changing the name:
AWS SDK Chime Meetings -> AWS Chime SDK Meetings endpoint

Reply:
Followed the similar pattern as "--chime-endpoint AWS SDK Chime endpoint"